pages.zh/common/asdf.md

```diff
@@ -1,28 +1,28 @@
 # asdf
 
-> 可扩展的包版本管理器，支持Nodejs,Ruby,Elixir,Erlang等.
-> 更多信息: <https://asdf-vm.com>.
+> 可扩展的包版本管理器，支持 Nodejs、Ruby、Elixir 和 Erlang 等。
```

Comment:
I found out that `、` is an "ideographic comma" (U+3001), but there's also the "halfwidth ideographic comma" (U+FF64) `､` and there's the "fullwidth comma" `，` (U+FF0C). Is there any rule when to use which? And what should we be using?

Reply:
According to [https://en.wikipedia.org/wiki/JIS_X_0201](https://en.wikipedia.org/wiki/JIS_X_0201), halfwidth ideographic comma derives from JIS X 0201, which is adopted by Japanese. So it is not suppose to appear in any Chinese page.

Reply:
Ideographic comma `、` is used when listing items, it's also called "enumeration comma". fullwidth comma `，` is used to join together clauses of certain topic.

Reply:
Interesting. For enumerating Latin text, should we use `、` or `,`? Currently both variants appear in the pages.

Reply:
I don't think I'm able to correct all the commas, since most of the time I have to understand the sentence to know if it's a enumeration or linking comma.

Reply:
Although using `，`  to list is weird for us, we still can understand what the sentence means. Listing without `、` is tolerable, we can fix it in future PRs.